extension/src/shared/view-layers/rootLayer.ts

```diff
@@ -170,18 +166,20 @@ const findEmphasisProperties = (
   featureInspector: FeatureInspectorSettings | undefined,
   templates: Template[],
   dataName: string | undefined,
+  dataSubName: string | undefined,
 ): EmphasisProperty[] | undefined => {
   const { useTemplate, templateId, properties } = featureInspector?.emphasisProperty ?? {};
 
+  if ((!useTemplate || !templateId) && !!properties?.length) return properties;
+
   // Default template
   const templateWithName = dataName
-    ? templates.find(t => t.name.split("/").slice(-1)[0] === dataName)
+    ? templates.find(
+        t =>
+          t.type === "emphasis" && [dataName, dataSubName].includes(t.name.split("/").slice(-1)[0]),
+      )
```

Comment:
Is default template for dataset still working?

Reply:
Yes, this part is for emphasis property only. Default field component template should be working also.